src/gui/vsWebSocket.h

```diff
@@ -65,16 +67,12 @@ class VsWebSocket : public QObject
     void disconnected();
 
    private slots:
-    void onConnected();
```

Comment:
I think `textMessageReceived` and `disconnected` should move in here because they are slots, not signals.

Reply:
I split this out into explicit signals/slots to make it less confusing. The previous implementation used a signal-to-signal connection.

Reply:
Sorry, I think I steered you wrong looking at your more recent changes. If the connections are meant to trigger new signals, I think you had it right the first time.

Reply:
I can switch it back